src/pages/graphql/schema/vault/queries/getVaultConfig.md

```diff
@@ -0,0 +1,104 @@
+---
+title: getVaultConfig query
+---
+
+# getVaultConfig query
+
+<InlineAlert variant="info" slots="text" />
+
+This query is available only if you have installed [Payment Services for Adobe Commerce](https://commercemarketplace.adobe.com/magento-payment-services.html) 2.10.0 or higher.
+
+The `getVaultConfig` query returns the vault configuration details for each payment method available. For now, only `credit_card` is supported.
+
+## Syntax
+
+```graphql
+{ 
+    getVaultConfig() VaultConfigOutput    
+}
+```
+
+## Example usage
+
+The following example runs the `getVaultConfig` query to retrieve all the stored payment details.
+
+**Request:**
+
+```graphql
+query {
+  getVaultConfig() {
+    credit_card {
+      is_vault_enabled
+      three_ds_mode
+      sdk_params {
+        name
+        value
+      }
+    }
+  }
+}
+```
+
+**Response:**
+
+```json
+{
+    "data": {
+        "getVaultConfig": {
+            "credit_card": {
+                "is_vault_enabled": true,
+                "three_ds_mode": "SCA_ALWAYS",
+                "sdk_params": [
+                    {
+                        "name": "src",
+                        "value": "https:\/\/www.paypal.com\/sdk\/js?client-id=..."
+                    },
+                    {
+                        "name": "data-partner-attribution-id",
+                        "value": "MagentoPayments_SP_PCP_Int"
+                    },
+                    {
+                        "name": "data-client-token",
+                        "value": "..."
+                    },
+                    {
+                        "name": "data-expires-in",
+                        "value": "3600"
+                    }
+                ]
+            }
+        }
+    }
+}
+```
+
+For now, only `credit_card` is supported with the `getVaultConfig` query.
```

Comment:
Delete

Reply:
why we want to delete this?. Right now, feature only works with `credit_card`

Reply:
silly me, I already mentioned earlier in the same page, thanks!